chrome/content/otr.js

```diff
@@ -276,6 +284,18 @@ var otr = {
     return human.readString();
   },
 
+  base64encode: function(data, dataLen) {
+    let buf = ctypes.char.array(Math.floor(((dataLen+2)/3)*4 + 1))();
+    let size = libOTR.otrl_base64_encode(buf, data, dataLen);
+    return buf.readString();
```

Comment:
According to the [doc](https://wiki.mozilla.org/Jsctypes/api):
> new t or new t() or t() - Create a new CData object of type t.
Without arguments, these allocate a new buffer of t.size bytes, populate it with zeroes, and return a new CData object referring to the complete object in that buffer.

Is it safe to make that assumption? I still confuse at this point.